zap_log_test.go

```diff
@@ -153,15 +162,16 @@ func (t *testZapLogSuite) TestLog(c *C) {
 		zap.Duration("duration", 10*time.Second),
 	)
 	lg.AssertMessage(
-		`[INFO] [zap_log_test.go:98] ["failed to fetch URL"] [url=http://example.com] [attempt=3] [backoff=1s]`,
-		`[INFO] [zap_log_test.go:103] ["failed to \"fetch\" [URL]: http://example.com"]`,
-		`[DEBUG] [zap_log_test.go:104] ["Slow query"] [sql="SELECT * FROM TABLE\n\tWHERE ID=\"abc\""] [duration=1.3s] ["process keys"=1500]`,
-		`[INFO] [zap_log_test.go:110] [Welcome]`,
-		`[INFO] [zap_log_test.go:111] ["Welcome TiDB"]`,
-		`[INFO] [zap_log_test.go:112] [欢迎]`,
-		`[INFO] [zap_log_test.go:113] ["欢迎来到 TiDB"]`,
-		`[WARN] [zap_log_test.go:114] [Type] [Counter=NaN] [Score=+Inf]`,
-		`[INFO] [zap_log_test.go:118] ["Testing typs"] [filed1=noquote] `+
+		`[INFO] [zap_log_test.go:113] ["failed to fetch URL"] [url=http://example.com] [attempt=3] [backoff=1s]`,
+		`[INFO] [zap_log_test.go:118] ["failed to \"fetch\" [URL]: http://example.com"]`,
+		`[DEBUG] [zap_log_test.go:119] ["Slow query"] [sql="SELECT * FROM TABLE\n\tWHERE ID=\"abc\""] [duration=1.3s] ["process keys"=1500]`,
+		`[INFO] [zap_log_test.go:125] [Welcome]`,
+		`[INFO] [zap_log_test.go:126] ["Welcome TiDB"]`,
+		`[INFO] [zap_log_test.go:127] [欢迎]`,
+		`[INFO] [zap_log_test.go:128] ["欢迎来到 TiDB"]`,
+		`[WARN] [zap_log_test.go:129] [Type] [Counter=NaN] [Score=+Inf]`,
+		`[INFO] [zap_log_test.go:133] ["new connection"],[connID=1] [traceID=dse1121]`,
```

Comment:
Why is there has a comma?

Reply:
it seems we should change this https://github.com/pingcap/log/pull/4/files#diff-22234ef75ee1f622759c3325ab0a28deR434